Integrations/DomainTools_Iris/iris.py

```diff
@@ -0,0 +1,393 @@
+    overall_risk_score = domain_result['domain_risk']['risk_score']
+    # The index 0 refers to "proximity" or "blacklist"
+    proximity_risk_score = domain_result['domain_risk']['components'][0]['risk_score']
+    domain_age = find_age(domain_result['create_date']['value'])
+    website_response = domain_result['website_response']
+    google_adsense = domain_result['adsense']['value']
+    google_analytics = domain_result['google_analytics']['value']
+    alexa_rank = domain_result['alexa']
+    tags = ', '.join(domain_result['tags'])
+
+    human_readable_data = {
+        "Overall Risk Score": overall_risk_score,
+        "Proximity Risk Score": proximity_risk_score,
+        "Domain Age (in days)": domain_age,
+        "Website Response": website_response,
+        "Google Adsense": google_adsense,
+        "Google Analytics": google_analytics,
+        "Alexa Rank": alexa_rank,
+        "Tags": tags
+    }
+
+    convert_empty_to_null(human_readable_data)
+
+    outputs = {}  # type: Dict[Any, Any]
+
+    headers = ["Overall Risk Score",
+               "Proximity Risk Score",
+               "Domain Age (in days)",
+               "Website Response",
+               "Google Adsense",
+               "Google Analytics",
+               "Alexa Rank",
+               "Tags"]
+    human_readable = tableToMarkdown('DomainTools Domain Analytics for {}.'.format(domain_result['domain']),
+                                     human_readable_data,
+                                     headers=headers)
+    return human_readable, outputs
+
+
+def format_threat_profile(response):
+    domain_result = response['results'][0]
+    domain_risk = domain_result['domain_risk']
+    overall_risk_score = domain_risk['risk_score']
+    threat_profile_data = get_threat_component(domain_risk['components'], 'threat_profile')
+    if threat_profile_data:
+        threat_profile_threats = ', '.join(threat_profile_data.get('threats', []))
+        threat_profile_evidence = ', '.join(threat_profile_data.get('evidence', []))
+    else:
+        threat_profile_threats = None
+        threat_profile_evidence = None
+
+    human_readable_data = {
+        "Overall Risk Score": overall_risk_score,
+        "Threat Profile Threats": threat_profile_threats,
+        "Threat Profile Evidence": threat_profile_evidence
+    }
+
+    outputs = {}  # type: Dict[Any, Any]
+
+    headers = ['Overall Risk Score', 'Threat Profile Threats', 'Threat Profile Evidence']
+    human_readable = tableToMarkdown('DomainTools Threat Profile for {}.'.format(domain_result['domain']),
+                                     human_readable_data,
+                                     headers=headers)
+    return human_readable, outputs
+
+
+def format_domain_pivot(response, search_type, search_value):
+    domain_list = [x['domain'] for x in response['results']]
+    human_readable_data = domain_list
+    domain_context_list = []
+
+    for domain_result in response['results']:
```

Comment:
There are two iterations over `response['results']`, no need for that, let's do it in once.